Infrastructure.DataAccess/Mapping/ItInterfaceMap.cs

```diff
@@ -14,6 +14,9 @@ public ItInterfaceMap()
             this.Property(x => x.Name)
                 .HasMaxLength(100) // http://stackoverflow.com/questions/1827063/mysql-error-key-specification-without-a-key-length
                 .IsRequired()
+                .HasUniqueIndexAnnotation("IX_NamePerOrg", 2);
+            this.Property(x => x.ItInterfaceId)
```

Comment:
ItInterfaceId må ikke være nullable, ellers kan duplikater opstå da null værdier ignoreres i indexet.
Dvs, at {Name: "1", ItInterfaceId: null} og {Name: "1", ItInterfaceId: null} er tilladt.

Når nullable feltet ændres husk at tage højde for nulls bliver til en tom streng og eksisterende værdier er uændret. Duplikater håndtere vi manuelt.